students/hyska_monika/lesson_11_argparse_math/selective_copy.py

```diff
@@ -0,0 +1,60 @@
+"""
+Program walks through a folder tree and searches for files
+with a certain file extension (such as .pdf or .jpg).
+Copy these files from whatever location they are in to a new folder.
+"""
+import shutil
+import os
+import argparse
+import sys
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Search files for selected'
+                                                 'extension and copy to chose '
+                                                 'location')
+    parser.add_argument('-p', '--path',
```

Comment:
You can also use positional arguments, which would be more natural here, similar to `cp` or `mv` commands.